src/core/services/theming/theme.palette.js

```diff
@@ -16,8 +16,7 @@ angular.module('material.core.theming.palette', [])
     'A400': '#ff1744',
     'A700': '#d50000',
     'contrastDefaultColor': 'light',
-    'contrastDarkColors': '50 100 200 300 A100',
-    'contrastStrongLightColors': '400 500 600 700 A200 A400 A700'
```

Comment:
Why `contrastStrongLightColors`? there are colors that the white or black doesn't have 87% opacity.